datamade/committee-oversight
# Add display names for committees to Wagtail

## Overview

Closes #174. This PR adds a `display_name` attribute to the committee detail pages. As committee names can shift slightly over Congress, but we'll want to retain each committee as a stable record, this will allow the Lugar Center to adjust committee names through the CMS.

## Testing Instructions

 * Download [this recent dump](https://s3.console.aws.amazon.com/s3/object/datamade-postgresql-backups/hearings/%255C2020%255C04%255C03%255C17%255C42.dump?region=us-east-1) from S3 and save it in the root of your project as `hearings.dump`
* Run `docker-compose down --volumes`
* Run `docker-compose -f docker-compose.yml -f docker-compose.db-ops.yml run -e PGPASSWORD=postgres --rm dbload-dump`
* Run `docker-compose run --rm app python manage.py migrate`
* Run `docker-compose up`
* Log into the site and edit a committee detail page. Change the `display name` at the bottom and make sure it shows up on the site, including on the landing page and hearing detail pages.
* Create a new committee page (you might have to delete an existing one). Make sure that it saves successfully without a modified display name.
